src/styles/_page.scss

```
@@ -1,3 +1,9 @@
+// Override for header height, possible theme option in the future
+body [class^='PageHeader-module--page-header'] {
+  height: rem(240px);
+  top: -9rem;
+}
+
```

Need to update how much the page header get's shifted so we don't shift the tabs off the screen.
```suggestion

[class^='PageHeader-module--page-header--shifted'] {
	top: 12rem;
}
```